creedengojava.properties

```diff
@@ -1,12 +1,19 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for JAVA language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/creedengo-java
-archivedVersions=2.0.0
-publicVersions=2.1.0
+archivedVersions=2.0.0,2.1.0
+publicVersions=2.1.1
 
 defaults.mavenGroupId=org.green-code-initiative
 defaults.mavenArtifactId=creedengo-java-plugin
 
+2.1.1.description=improve library versions; check and upgrades for SonarQube until 25.3.0 version
+2.1.1.sqcb=[24.12,25.3.*]
```

Comment:
Did you mean to not have any compatibility with SonarQube Server?

Reply:
Hi @ganncamp, 
thank you for your feedback.
But, no, it isn't what I try to say ... sorry for mis-understanding.
We checked that our plugin is compatible with all SonarQube server version until the last one, 25.3.0.
We changed internally our tested SonarQube version (sonarqube docker version, and so on ...)
I don't understand what is wrong with my PR.

Reply:
correction done with LATEST